docs/troubleshooting-ios-build.md

```
@@ -0,0 +1,73 @@
+   VALID_ARCHS = "arm64 x86_64";
+   ```
+
+3. **Update EXCLUDED_ARCHS**:
+   
+   Replace:
+   ```
+   "EXCLUDED_ARCHS[sdk=iphoneos*]" = "arm64-sim x86_64";
+   ```
+   
+   With:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)